HeatApp/components/screens/StartSettingsScreen.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import {
+  Text, StyleSheet, Dimensions, TouchableOpacity
+} from "react-native";
+
+const {width: WIDTH} = Dimensions.get('window');
+
+export default class LoginScreen extends React.Component {
+  _start = () => {
+    this.props.navigation.navigate("Main");
+  };
+
+  constructor() {
+    super();
+  }
+
+  render() {
+    return (
+      <TouchableOpacity style={styles.startBtn}>
+        <Text style={styles.text} onPress={this._start}> Start </Text>
+      </TouchableOpacity>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  startBtn: {
+    width: WIDTH - 55,
+    height: 45,
+    borderRadius: 25,
+    backgroundColor: "rgba(21, 211, 102, 0.8)",
```

Comment:
Same for color and all other colors in this PR. It's better to store them in a single file as constants. Background color can be hex, can't it?

Reply:
Yes, of course it can be) It's just example))